Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -2408,6 +2412,9 @@ def qradar_search_create_command(client: Client, params: Dict, args: Dict) -> Co
                                         int(offense_id),
                                         start_time,
                                         return_raw_response=True)
+        if response == QueryStatus.ERROR.value:
+            raise DemistoException(f'Could not create search for offense_id: {offense_id}')
```

Comment:
```suggestion
            raise DemistoException(f'Could not create events search for offense_id: {offense_id}')
```
Distinguish the exception and log messages.